packages/api/endpoints/rules.js

```diff
@@ -94,30 +95,47 @@ async function put(event) {
     originalData = await model.get({ name });
   }
   catch (e) {
-    if (e instanceof RecordDoesNotExist) {
-      throw new Error({ message: 'Record does not exist' });
-    }
+    if (e instanceof RecordDoesNotExist) return cb({ message: 'Record does not exist' });
+    return cb(e);
   }
 
   // if rule type is onetime no change is allowed unless it is a rerun
   if (action === 'rerun') {
     await models.Rule.invoke(originalData);
-    return;
+    return cb(originalData);
```

Comment:
I think this should be `return cb(null, originalData);`

Reply:
Yes.  Updated.